indicators/tuya-redirect-4ewqnc.yml

```diff
@@ -6,33 +6,26 @@ description: |
 
 references:
     - https://www.tuya.com.co/
-    - https://urlscan.io/result/026d1d48-ba58-477d-a168-0581ef8b9ebc/
+    - https://urlscan.io/result/630122d4-6b1b-4b56-bb13-6b6a1f28af4f/
 
 detection:
 
     title:
-      html|contains:
-        - Portal Transaccional
+      html|contains: Portal Transaccional
+
+    scriptSource:
+      html|contains: 'src="https://us-sonar.sociomantic.com/js/2010-07-01/adpan/tuya-co"'
```

Comment:
It seems this script is used by the legitimate website as well, so I would suggest using the following string instead.

```
insertclavex
```